lib/diagnostics.cpp

```diff
@@ -496,7 +500,7 @@ int diagnostics_init(
         //   that for future use.
         lReturnValue = RegOpenKeyEx(
             HKEY_LOCAL_MACHINE,
-            _T("SOFTWARE\\Space Sciences Laboratory, U.C. Berkeley\\BOINC Setup"),
+            TEXT("SOFTWARE\\Space Sciences Laboratory, U.C. Berkeley\\BOINC Setup"),
```

Comment:
`_T` correctly resolves to Unicode if needed.
`TEXT` by default is just a string literal as-is.
What is the reason of this change?

Reply:
API correctness
`TEXT` is the generic-text macro for the Windows API
`_T` is for the CRT